doc/ref_model/chapters/chapter03.md

```diff
@@ -49,15 +49,15 @@
   * [3.8.8 Smart Switches](#3.8.8)
   * [3.8.9 Decoupling Applications from Infrastructure and Platform with Hardware Acceleration](#3.8.9)
 
-It is necessary to clearly define the infrastructure resources and their capabilities a shared cloud infrastructure (network function virtualisation infrastructure, NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNFs). The lack of a common understanding of which resources and corresponding capabilities a suitable cloud infrastructure should provide may lead to several issues which could negatively impact the time and the cost for on-boarding and maintaining these solutions on top of a virtualised infrastructure.
+It is necessary to clearly define the infrastructure resources and their capabilities that a shared cloud infrastructure (network function virtualisation infrastructure, NFVI) will provide for hosting workloads including virtual network functions (VNFs) and/or cloud-native network functions (CNFs). A common understanding of which resources and corresponding capabilities a suitable cloud infrastructure should provide will help to improve workload on-boarding efficency and avoid issues which could negatively impact the time and the cost of on-boarding and maintaining target workloads and solutions on top of a virtualised infrastructure.
```

Comment:
@jbhartley re: "... which resources and corresponding capabilities a suitable cloud infrastructure **should provide** ..." (emphasis mine). Do we want to say "provides" because if the cloud infrastructure doesn't provide the necessary resources then onboarding may not be feasible?

Reply:
@pgoyal01  Your feedback is valid.

The first couple of sentences are very hard reading, I re-read them a few time to try to understand "meaning".

I am also concerned on the definition of "Tenant":
3.2.1.1 Tenant
A cloud infrastructure needs to be capable of supporting multiple tenants and has to isolate sets of infrastructure resources dedicated to specific workloads (VNF/CNF) from one another. Tenants represent an independently manageable logical pool of compute, storage and network resources abstracted from physical hardware.

I believe that what we are defining is a "tenancy" not a "tenant" but his is likely an item for discussion subsequent to "proof reading".

Reply:
@pgoyal01 and @jbhartley: regarding comment from @pgoyal01: I think that we need both "should provide and actually provides".  The former addresses the need to understand the requirements while the latter - that these requirements are met prior to on-boarding.

Reply:
@jbhartley: you are right about tenant vs tenancy.  But I also agree we should leave it for the next release (worthwhile raising an Issue in GitHub on this), as this is more than proofreading.